test/analytics.test.js

```diff
@@ -409,7 +409,6 @@ describe('Analytics', function() {
       var a = new Identify({ userId: 'id', traits: { trait: true } });
       analytics._invoke('identify', a);
       var b = Test.prototype.invoke.args[0][1];
-      assert(b === a);
```

Comment:
Rationale: We're now making a copy, so these references are no longer identical.